images/macos/provision/core/android-toolsets.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -e -o pipefail
 source ~/utils/utils.sh
+source ~/utils/invoke-tests.sh
```

Comment:
There is one small function in this file, I believe it can be moved to `utils.sh`

Reply:
We have a plan to do this trick: https://github.com/actions/virtual-environments/pull/2270/files#r544082173
In this case, it won't be needed to import it everywhere

Reply:
ok, thanks!